contribs/gnofaucet/serve.go

```diff
@@ -95,7 +99,7 @@ func (c *serveCfg) RegisterFlags(fs *flag.FlagSet) {
 	fs.StringVar(
 		&c.mnemonic,
 		"mnemonic",
-		"",
+		os.Getenv("GNO_MNEMONIC"),
```

Comment:
This shouldn't exist (env reading). It needs to be explicitly set as part of the cmd run

Reply:
removed here 82c3600f317e8a29df77f24290a24298d7332314